CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(backtrace)

```

Comment:
backtrace or libbacktrace?  People usually say libbacktrace, and that is what autoconf uses.  Should probably libbacktrace unless there is some reason not to.

Reply:
I suggest to use `libbacktrace`, that way it will be different from the [FindBacktrace](https://cmake.org/cmake/help/latest/module/FindBacktrace.html) CMake module associated with [backtrace(3)](https://linux.die.net/man/3/backtrace)